examples/camunda-spring-boot-starter-example-simple/src/main/java/org/camunda/bpm/spring/boot/starter/example/simple/Showcase.java

```diff
@@ -5,9 +5,9 @@
 import org.camunda.bpm.engine.RuntimeService;
 import org.camunda.bpm.engine.TaskService;
 import org.camunda.bpm.engine.task.Task;
+import org.camunda.bpm.spring.boot.starter.events.ProcessApplicationStartedEvent;
```

Comment:
We should stick to singular for package names ... so starter.event would be my choice